README.md

```diff
@@ -43,23 +43,57 @@ curl 'http://localhost:5000/parse?q=hello'
 ### B. Install with Docker
 Before you start, ensure you have the latest version of docker engine on your machine. You can check if you have docker installed by typing ```docker -v``` in your terminal.
 
-#### 1. Build the image:
+#### B.1 Install with Docker manually
+##### 1. Build the image:
 ```
 docker build -t rasa_nlu .
 ``` 
 
-#### 2. Start the web server:
+##### 2. Start the web server:
 ```
 docker run -p 5000:5000 rasa_nlu start
 ```
 
 Caveat for Docker for Windows users: please share your C: in docker settings, and add ```-v C:\path\to\rasa_nlu:/app``` to your docker run commands for download and training to work correctly.
 
-#### 3. Test it!
+##### 3. Test it!
 ```
 curl 'http://localhost:5000/parse?q=hello'
 ```
 
+#### B.2 Install with Docker automatically
+Automatically installation uses spacy english model and configure trained model as default if there is only one trained model.
+If you want to use another model read the Dockerfile, comment spacy english model lines and uncomment the lines for your selected model.
+
+##### 1. Create container
+
+Execute as root:
+```
+chmod +x ./manage_container
```

Comment:
He is talking about the `manage_container` file you added. You can execute can execute `chmod +x ./manage_container` locally and comit the file with the changed permissions to github (github should persist these permissions as well) - hence, you can remove this line after you changed and added the file with the updated permissions to github.